spec/models/publication_spec.rb

```diff
@@ -332,21 +329,6 @@
     end
   end
 
-  describe 'contributions.build_or_update' do
-    it 'should add a contribution' do
-      c = publication.contributions.build_or_update(author, status: 'unknown')
-      expect(c.author).to eq(author)
-      expect(c.status).to eq('unknown')
-    end
-
-    it 'should update a contribution record if the association exists' do
-      publication.contributions.build_or_update author
-      c = publication.contributions.build_or_update(author, status: 'new')
-      expect(c.author).to eq(author)
-      expect(c.status).to eq('new')
-    end
-  end
```

Comment:
If it's not in the method any more, the equivalent behavior still needs to be tested somehow.

Reply:
This is covered by the API tests.

Reply:
OK, didn't see that right away.